src/Sql/Sql/help/Clear-AzSqlDatabaseVulnerabilityAssessmentRuleBaseline.md

```diff
@@ -107,7 +110,8 @@ Accept wildcard characters: False
 ```
 
 ### -RuleAppliesToMaster
-Specifies whether the baseline results should apply on a server level rule identified by the RuleId
+Specifies whether the baseline results should apply on a server level rule identified by the RuleId.
```

Comment:
Specifies that the baseline should apply to the master database.
This is required only when database level settings specify a storage account that is different from the one specified in the serverl level settings.